securityconfig/roles.yml

```diff
@@ -69,27 +69,3 @@ reports_instances_read_access:
     - 'cluster:admin/opendistro/reports/instance/list'
     - 'cluster:admin/opendistro/reports/instance/get'
     - 'cluster:admin/opendistro/reports/menu/download'
-
-# Allows users to read and download Reports and Report-definitions
```

Comment:
I believe each individual permission like 'cluster:admin/opendistro/reports/definition/create' ( pattern:  'cluster:admin/opendistro/reports/*' )  is no longer defined. 

Otherwise one can still create its own custom role with these permission to use this feature.

Reply:
https://github.com/opendistro-for-elasticsearch/security-kibana-plugin/pull/657 is removing the option to create from user.